saleor/checkout/utils.py

```diff
@@ -929,6 +962,7 @@ def get_shipping_price_estimate(checkout: Checkout, discounts, country_code):
     if shipping_methods is None:
         return None
 
+    # TODO: extension manager should be able to have impact on shipping price estimates
```

Comment:
Maybe we could create a method for that or gh issue instead of `TODO`?

Reply:
Done #5001